### PR TITLE
Replace branded consent journey

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
The branded version of the consent journey is no longer needed for Friday (at least temporarily).  This PR just replaces it with the "regular" journey, leaving the functionality intact in case we get updated branding going forwards (all TBD).

Looks like there are some changes here that should have been in the last PR instead, but they don't affect functionality anyway - just dead code being removed.

### CODE REVIEW

**Review by rahilb — APPROVED** [review]
LGTM

File: identity/app/views/consentJourney.scala.html [hunk 1]
```diff
@@ -133,7 +133,7 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 @introStep = @{
     ConsentStep(
         name = "intro",
-        title = "Please select the emails you wish to continue receiving",
+        title = "Please select the emails you wish to receive",
```
Comment (walaura) [inline identity/app/views/consentJourney.scala.html]: 😌

File: identity/conf/routes [hunk 2]
```diff
@@ -71,7 +71,7 @@ GET         /adverts/manage                         controllers.AdvertsManager.r
 # Consents journey
 ########################################################################################################################
 GET         /consents/thank-you                     controllers.editprofile.EditProfileController.displayConsentsJourneyThankYou
-GET         /consents/staywithus                    controllers.editprofile.EditProfileController.displayConsentsJourneyGdprCampaign
+GET         /consents/staywithus                    controllers.editprofile.EditProfileController.displayConsentsJourney(consentHint: Option[String])
```
Comment (walaura) [inline identity/conf/routes]: could this be a redirect instead? (like `displayPrivacyFormRedirect`) Even if we do update the branding this URL no longer makes sense

**Review by walaura — APPROVED** [review]
LGTM!

That said I personally advocate for no longer branding this page (any further branding should be part of our new signup strategy matrix thing and not use `/consents`)

**Comment by jfsoul** [comment]
@walaura Agree with you, just put this in as a very temporary quick fix.  I think we should revisit this next week which is why I put in a TODO comment as a reminder.

**Pull request merged**